server/server.go

```diff
@@ -16,6 +17,13 @@ import (
 )
 
 type cribbageServer struct {
+	dbService persistence.DB
```

Comment:
🤔 I'm not sure how I feel about calling this a `dbService`. Can we just call it a `db persistence.DB` since that what the package calls it?